src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -110,14 +106,17 @@ class SparqlQleverVisitor {
   // meaning of blank and anonymous nodes is different.
   bool isInsideConstructTriples_ = false;
 
+  // Resets the Visitors state between updates. This resets everything except
+  // prefix and base.
```

Comment:
Please also add a bit comment with a `NOTE:` at the beginning and the end of the data members, that one 
has to typically revisit the `resetState` function when adding data members to the UPDATE.